Voycar.Api.Web.Tests.Integration/Users/Post_Login/Tests.cs

```diff
@@ -0,0 +1,206 @@
+            HouseNumber = "null",
+            PostalCode = "null",
+            City = "null",
+            Country = "null",
+            BirthDate = new DateOnly(2002, 12, 12),
+            BirthPlace = "null",
+            PhoneNumber = "null",
+            PlanId = this._state.PlanId
+        };
+    }
+
+    [Fact]
+    public async Task Post_Request_ReturnsBadRequest_DueToUnverifiedUser()
+    {
+        // Arrange
+        var request = new R.Request { Email = "unverified@test.de", Password = "notsafe987" };
+
+        await this._app.Client
+            .POSTAsync<Features.Users.Endpoints.Post.Register.Endpoint, Features.Users.Endpoints.Post.Register.Request>(
+                this.CreateValidRequest()); // Register new user without verifying
+
+        // Act
+        var httpResponse = await this._app.Client.POSTAsync<R.Endpoint, R.Request>(request);
+
+        // Assert
+        httpResponse.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+
+
+    [Fact]
+    public async Task Post_Request_ReturnsBadRequest_DueToIncorrectPassword()
+    {
+        // Arrange
+        var memberClient =
+            await ClientFactory.CreateMemberClient(this._app, this.Context, "member@test.de", "password");
+        var request = new R.Request { Email = "member@test.de", Password = "diffPassword" };
+
+        // Act
+        var firstHttpResponse = await memberClient.PostAsync("/auth/logout", default); // Logout
+        var secondHttpResponse = await this._app.Client.POSTAsync<R.Endpoint, R.Request>(request); // Login
+
+        // Assert
+        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+        secondHttpResponse.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+
+
+    [Fact]
+    public async Task Post_Request_ReturnsBadRequest_DueToNonExistingUser()
+    {
+        // Arrange
+        var client = this._app.CreateClient();
+        var request = new R.Request { Email = "doesNotExist@test.de", Password = "diffPassword" };
+
+        // Act
+        var httpResponse = await client.POSTAsync<R.Endpoint, R.Request>(request);
+
+        // Assert
+        httpResponse.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+
+
+    [Fact]
+    public async Task Post_Request_ReturnsOk_And_MemberIsLoggedIn()
+    {
+        // Arrange
+        var memberClient =
+            await ClientFactory.CreateMemberClient(this._app, this.Context, "memberClient@test.de", "password");
+        var request = new R.Request { Email = "memberClient@test.de", Password = "password" };
+
+        // Act
+        var firstHttpResponse = await memberClient.PostAsync("/auth/logout", default); // Logout
+        var secondHttpResponse = await this._app.Client.POSTAsync<R.Endpoint, R.Request>(request); // Login
+        var thirdHttpResponse = await this._app.Client.GetAsync("/user/whoami"); // Test if login was successful
+
+        // Assert
+        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
```

Comment:
Ähnlich wie beim anderen Test
```suggestion
        firstHttpResponse.StatusCode.Should().Be(HttpStatusCode.OK,
            "factory method produced logged in user who should be able to log out");
```